ubuntu/small-ubuntu-24.04.pkr.hcl

```diff
@@ -21,41 +21,38 @@ variable "SSH_PUB_KEY" {
   default = ""
 }
 
-source "qemu" "small_ubuntu_20_04" {
+source "qemu" "small_ubuntu_24_04" {
   boot_command              = [
-    "<enter><enter><f6><esc><wait>",
-    "<bs><bs><bs><bs><bs><bs><bs><bs>",
-    "<bs><bs><bs><bs><bs><bs><bs><bs>",
-    "<bs><bs><bs><bs><bs><bs><bs><bs>",
-    "<bs><bs><bs><bs><bs><bs><bs><bs>",
-    "<bs><bs><bs><bs><bs><bs><bs><bs>",
-    "/casper/vmlinuz ",
-    "initrd=/casper/initrd ",
-    " autoinstall ds=nocloud-net;s=http://{{ .HTTPIP }}:{{ .HTTPPort }}/ ",
+    "c<wait>",
+    "linux /casper/vmlinuz --- autoinstall ds=\"nocloud-net;seedfrom=http://{{ .HTTPIP }}:{{ .HTTPPort }}/\"",
     " PACKER_USER=ubuntu PACKER_AUTHORIZED_KEY={{ `${var.SSH_PUB_KEY}` | urlquery }}",
-    "<wait><enter>"
+    "<enter><wait>",
+    "initrd /casper/initrd",
+    "<enter><wait>",
+    "boot",
+    "<enter>"
   ]
-  boot_wait                 = "3s"
-  disk_size                 = 3500
+  boot_wait                 = "6s"
+  disk_size                 = 4000
   format                    = "qcow2"
   headless                  = true
   http_directory            = "httpdir"
   http_port_max             = 8550
   http_port_min             = 8500
-  iso_checksum              = "sha256:b8f31413336b9393ad5d8ef0282717b2ab19f007df2e9ed5196c13d8f9153c8b"
-  iso_url                   = "https://releases.ubuntu.com/20.04/ubuntu-20.04.6-live-server-amd64.iso"
+  iso_checksum              = "sha256:d6dab0c3a657988501b4bd76f1297c053df710e06e0c3aece60dead24f270b4d"
+  iso_url                   = "https://releases.ubuntu.com/24.04/ubuntu-24.04.2-live-server-amd64.iso"
   memory                    = 1024
   qemuargs                  = [["-cpu", "host"]]
   shutdown_command          = "sudo -- sh -c 'rm /etc/sudoers.d/99-egi-installation && shutdown -h now'"
   ssh_clear_authorized_keys = true
   ssh_private_key_file      = "${var.SSH_PRIVATE_KEY_FILE}"
   ssh_timeout               = "20m"
   ssh_username              = "ubuntu"
-  vm_name                   = "Small.Ubuntu.20.04-2024.11.18"
+  vm_name                   = "Small.Ubuntu.24.04-2025.05.09"
```

Comment:
minor comment, but this name is not unique. If concurrent builds occur it will create problems. Suggest adding a uuid to the name as suffix